assets/demo-todo-app.spec.js

```diff
@@ -81,7 +81,7 @@ test.describe('Mark all as completed', () => {
 
   test('should allow me to mark all items as completed', async ({ page }) => {
     // Complete all todos.
-    await page.locator('.toggle-all').check();
+    await page.getByText('Mark all as complete').check();
```

Comment:
```suggestion
    await page.getByLabel('Mark all as complete').check();
```

lets use getByLabel instead and below, filed https://github.com/microsoft/playwright/issues/18297 for it.